packages/components/src/external-link/index.tsx

```diff
@@ -9,13 +9,11 @@ import type { ForwardedRef } from 'react';
  */
 import { __ } from '@wordpress/i18n';
 import { forwardRef } from '@wordpress/element';
-import { external } from '@wordpress/icons';
```

Comment:
I do favor the idea of not relying on icons when we don't need to. It sure helps keep things lighter.